docs/2.0/integration/testng.adoc

```diff
@@ -19,7 +19,7 @@ You need to add the following to your **pom.xml**:
     <compiler.plugin.version>3.6.0</compiler.plugin.version>
     <java.version>1.8</java.version>
     <surefire.plugin.version>2.19.1</surefire.plugin.version>
-    <allure.version>2.0-SNAPSHOT</allure.version>
+    <allure.version>2.0-BETA6</allure.version>
```

Comment:
@L-Evg that's a question to @baev @eroshenkoam. Command-line is released, however adapters are still in beta.